src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -143,6 +144,9 @@
 
       <SourcesToCopyToTestDir Include="@(ContentWithTargetPath)" />
```

Comment:
@ellismg @MattGal Any idea why this does't contain TestCopyLocal? I'm been trying to read through this logic and I must say I'm kind of lost. I think this file needs a bit of an overhaul because there seems to be some dead ends and duplication (and duplication removal :)). I don't even think we are using the CreateTestNuGetPackage for anything and it is the reason for some of this duplicate item groups. I'm also unclear why we are still calling Copy on these items when the RunTests.cmd is supposed to be the one doing the copying now. There is too much that is unclear to me right now without actually debugging through this to see what is happening.